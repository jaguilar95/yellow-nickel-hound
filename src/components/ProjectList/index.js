import React from "react";

function ProjectList() {
  const photos = [
    {
      name: "Voting with Friends",
      description: "Create polls and share with friends",
      docName: "VotingWithFriends.png",
      github:
        "https://github.com/jaguilar95/thejake989-PurpleAluminiumShih-Tzu",
    },
    {
      name: "Budget Tracker",
      description: "Track your expenses on the go",
      docName: "secure-sierra-42105.herokuapp.com_.png",
      github: "https://github.com/jaguilar95/navy-bronze-terrier",
    },
    {
      name: "Note Taker",
      description: "Dynamically create notes saved locally",
      docName: "aqua-iron-dane.herokuapp.com_(iPad Air).png",
      github: "https://github.com/jaguilar95/aqua-iron-dane",
    },
    {
      name: "Deck of Many Things",
      description: "Randomly draw from the Deck of Many Things",
      docName: "127.0.0.1_5500_domt-app_index.html(iPad Air).png",
      github: "https://github.com/tj-tieso/domt-app",
    },
    {
      name: "Weather Dashboard",
      description: "Get local 5-day weather forecast",
      docName: "127.0.0.1_5500_index.html_.png",
      github: "https://github.com/jaguilar95/white-mercury-dalmatian",
    },
    {
      name: "Work Day Scheduler",
      description: "A simple calendar for scheduling your work day",
      docName: "jaguilar95.github.io_gray-brass-spaniel_.png",
      github: "https://github.com/jaguilar95/gray-brass-spaniel",
    },
  ];
  return (
    <section>
      <h1>Portfolio</h1>
      <div className="flex-row">
        {photos.map((image) => (
          <a href={image.github}>
            <img
              src={require(`../../assets/ProjectList/${image.docName}`)}
              alt={image.description}
              className="img-thumbnail mx-1"
              key={image.name}
            />
          </a>
        ))}
      </div>
    </section>
  );
}

export default ProjectList;
