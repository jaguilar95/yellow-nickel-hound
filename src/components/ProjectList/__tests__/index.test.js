import React from "react";
import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import ProjectList from "..";

afterEach(cleanup);

describe("ProjectList/Portfolio", () => {
  it("renders", () => {
    render(<ProjectList />);
  });

  it("project list render matched snapshot", () => {
    const { asFragment } = render(<ProjectList />);

    expect(asFragment()).toMatchSnapshot();
  });
});
