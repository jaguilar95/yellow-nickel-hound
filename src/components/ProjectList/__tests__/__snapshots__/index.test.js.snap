// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectList/Portfolio project list render matched snapshot 1`] = `
<DocumentFragment>
  <section>
    <h1>
      Portfolio
    </h1>
    <div
      class="flex-row"
    >
      <a
        href="https://github.com/jaguilar95/thejake989-PurpleAluminiumShih-Tzu"
      >
        <img
          alt="Create polls and share with friends"
          class="img-thumbnail mx-1"
          src="VotingWithFriends.png"
        />
      </a>
      <a
        href="https://github.com/jaguilar95/navy-bronze-terrier"
      >
        <img
          alt="Track your expenses on the go"
          class="img-thumbnail mx-1"
          src="secure-sierra-42105.herokuapp.com_.png"
        />
      </a>
      <a
        href="https://github.com/jaguilar95/aqua-iron-dane"
      >
        <img
          alt="Dynamically create notes saved locally"
          class="img-thumbnail mx-1"
          src="aqua-iron-dane.herokuapp.com_(iPad Air).png"
        />
      </a>
      <a
        href="https://github.com/tj-tieso/domt-app"
      >
        <img
          alt="Randomly draw from the Deck of Many Things"
          class="img-thumbnail mx-1"
          src="127.0.0.1_5500_domt-app_index.html(iPad Air).png"
        />
      </a>
      <a
        href="https://github.com/jaguilar95/white-mercury-dalmatian"
      >
        <img
          alt="Get local 5-day weather forecast"
          class="img-thumbnail mx-1"
          src="127.0.0.1_5500_index.html_.png"
        />
      </a>
      <a
        href="https://github.com/jaguilar95/gray-brass-spaniel"
      >
        <img
          alt="A simple calendar for scheduling your work day"
          class="img-thumbnail mx-1"
          src="jaguilar95.github.io_gray-brass-spaniel_.png"
        />
      </a>
    </div>
  </section>
</DocumentFragment>
`;
